express-delete/index.js

```diff
@@ -0,0 +1,35 @@
+const express = require('express');
+const userRequest = express.json();
+const app = express();
+
+const grades = [
+  {
+    id: 1,
+    name: 'Beeg Yoshi',
+    course: 'Meming',
+    grade: 9001
+  },
+  {
+    id: 2,
+    name: 'Smol Mario',
+    course: 'Jumping',
+    grade: 9004
+  }
+];
+
+app.use(userRequest);
+
+app.get('/api/grades', (req, res) => {
+  res.json(grades);
+});
+
+app.delete('/api/grades/:gradesIndex', (req, res) => {
+  const requestedIndexValue = req.params.gradesIndex - 1;
```

Comment:
The index and id of an object are not related. the order may change or the middle element of (say 5) may be deleted from the array. Use a loop instead to search the array.